src/main/java/com/example/realworld/model/ProfileResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.example.realworld.model;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import lombok.Builder;
+import lombok.Data;
+
+@JsonTypeName("profile")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.WRAPPER_OBJECT)
+@Data
+@Builder
+public class ProfileResponse {
```

Comment:
```suggestion
public class ProfileResDto {
```
Chuyển về package `dto`